InvenTree/common/notifications.py

```diff
@@ -8,15 +8,19 @@
 from InvenTree.helpers import inheritors
 from InvenTree.ready import isImportingData
 from common.models import NotificationEntry, NotificationMessage
+from common.models import InvenTreeUserSetting
+
 import InvenTree.tasks
 
 
 logger = logging.getLogger('inventree')
 
 
-# region notification classes
```

Comment:
I would advocate to leave the regions in. They help to structure code in a bigger file if opened with a lot of bigger IDEs like pycharm, visual studio or visual studio code (with a plugin, the bultin function was removed a few releases ago)